### PR TITLE
Adding Git submodules, fix Bicep module references, update installati…

### PR DESCRIPTION
**Pull request by pauldotyu**
…on instructions, and add devcontainer configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,56 @@
...
 ```
 
+> :bulb: If you have access to multiple subscriptions, you may need to run the following command to work with the appropriate subscription:
+>
+> `az account set --subscription <NAME_OR_ID_OF_SUBSCRIPTION>`
+
 Install kubectl using the Azure CLI, if required.
 
 ```bash
 az aks install-cli
```
Comment (jgardner04) [inline README.md:29]: I had to use `sudo` to get this to install on the DevContainer. Should we pre-install this in the `installTools.sh`?

**Review by jgardner04 — APPROVED** [review]
I have an issue because I am using a newer version of Kustomize locally. I used the DevContainer and was able to get everything to deploy. I did add a note that I had to use `sudo` to get some of the tools deployed appropriately in the DevContainer but I don't think this is a blocker.

**Comment by mosabami** [comment]
thanks for the review @jgardner04

File: README.md (hunk 1 above)
Comment (pauldotyu) [reply README.md:29]: In the devcontainer, I am including the `ghcr.io/devcontainers/features/kubectl-helm-minikube:1` feature which includes kubectl, so this is not really needed. I can update the README.md to say something like, "If you don't already have kubectl installed, you can install it using this command"

**Pull request merged**